{
	"id": 220682400,
	"body": "Your patch looks fixes the issue (it no longer occurs on the small \u0026 large memory case).\r\n\r\nI've also modified the test to better reflect what the real use case is - https://gist.github.com/nemothekid/be66f5b4521fbe48cf775a18783ea7d7 - this is when the server sees the request is too large, and refuses to read it (again, it passes it with your patch, but not with `CloseWithError`).\r\n\r\nThis uses values large enough to fail reliably for me, but not use a huge amount of memory. Note that where I have the `log.Fatalf` is the test, I don't really care about the response, only that the request shouldn't fail with an internal error.\r\n\r\n\r\nAlso, with that same script I am using:\r\n\r\n* `go version go1.6.2 darwin/amd64`\r\n* `golang.org/x/net/http2` at the current master.\r\n",
	"user": {
		"login": "nemothekid",
		"id": 940094,
		"type": "User",
		"site_admin": false
	},
	"created_at": "2016-05-20T18:24:54Z",
	"updated_at": "2016-05-20T18:28:36Z"
}
